src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,108 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final double INCREASE_COEFFICIENT = 1.5;
+    private static final int DEFAULT_CAPACITY = 10;
+    private int capacity;
```

Comment:
you don't need this field, remove it and rewrite your logic, so you won't need such one